src/Compiler/Service/service.fsi

```diff
@@ -269,6 +272,7 @@ type public FSharpChecker =
     member GetProjectSnapshotFromScript:
```

Comment:
This is part of the public surface area, is it not?
Is always passing "None" equivalent to old behavior? We should speak with Rider and VsCode to explain how and why to use it.